packages/theming/src/utils/getLineHeight.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import stripUnit from 'polished/lib/helpers/stripUnit';
+
+/**
+ * Get unitless line height based on the given pixel-valued height and font size.
+ *
+ * @param {string|number} height Desired line height in pixels.
+ * @param {string|number} fontSize Font size (in pixels) of text contained within the line.
+ *
+ * @component
+ */
+export default function getLineHeight(height: string | number, fontSize: string | number) {
```

Comment:
Should we be using named exports for utilities?

Reply:
I'm currently following the style used for utilities since package inception. If we decide to update, let's make them all consistent under a separate PR.